### PR TITLE
Feature/intro

### PR DESCRIPTION
**Pull request by yosser**

### CODE REVIEW

**Comment by andgreendigital** [comment]
I'll review this after #60 is merged

**Comment by andgreendigital** [comment]
Can you merge develop into this branch?

File: src/components/intro.jsx [hunk 1]
```diff
@@ -3,32 +3,24 @@ import PropTypes from 'prop-types';
...
-    >
-        <h1 className={css.intro_title}>
-            {title}
-        </h1>
-        <h2 className={css.intro_text}>
-            {tagline}
-        </h2>
+    <section className={css.intro__section} style={{ background: `linear-gradient(to bottom left, ${hexCodeTopRight}, ${hexCodeBottomLeft})` }}>
+        <div className={`${css.intro__section__box} ${css.intro__section__left}`}>
+            <h3 className={`font_style__B1 ${css.intro__section__breadcrumbs}`}>{breadcrumbs}</h3>
```
Comment (andgreendigital) [inline src/components/intro.jsx]: Should we use the breadcrumbs component here?

Comment (yosser) [reply src/components/intro.jsx]: I have used it - since the merge

File: src/components/intro.jsx [hunk 2]
```diff
@@ -1,34 +1,28 @@
...
-    >
-        <h1 className={css.intro_title}>
-            {title}
-        </h1>
-        <h2 className={css.intro_text}>
-            {tagline}
-        </h2>
+    <section className={css.intro__section} style={{ background: `linear-gradient(to bottom left, ${hexCodeTopRight}, ${hexCodeBottomLeft})` }}>
+        <div className={`${css.intro__section__box} ${css.intro__section__left}`}>
+            <h3 className={`font_style__B1 ${css.intro__section__breadcrumbs}`}><Breadcrumbs home={home} pathname={pathname} /></h3>
```
Comment (andgreendigital) [inline src/components/intro.jsx]: Don't think this `h3` is needed. I think the breadcrumb component manages its own styling are styled already.

Comment (yosser) [reply src/components/intro.jsx]: Not any more it doesnt

Comment (andgreendigital) [reply src/components/intro.jsx]: What do you mean? Did you remove styling from the breadcrumb?

File: src/components/breadcrumbs.jsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ const Breadcrumbs = ({ pathname, home }) => {
             {home && (<Link to="/" className={css.breadcrumb_link}>Home / </Link>)}
             {
                 breadcrumbs.map((crumb) => (crumb.label ? (
-                    <Link to={crumb.slug} className={css.breadcrumb_link}>
+                    <Link key={crumb.slug} to={crumb.slug} className={css.breadcrumb_link}>
```
Comment (eugene-matvejev) [inline src/components/breadcrumbs.jsx]: 👍

File: src/components/breadcrumbs.jsx [hunk 4]
```diff
@@ -37,7 +37,7 @@ const Breadcrumbs = ({ pathname, home }) => {
 
 Breadcrumbs.propTypes = {
     pathname: PropTypes.string.isRequired,
-    home: PropTypes.string,
+    home: PropTypes.bool,
```
Comment (eugene-matvejev) [inline src/components/breadcrumbs.jsx]: 👍

File: src/templates/blog-post-template.jsx [hunk 5]
```diff
@@ -210,20 +210,7 @@ const BlogPostTemplate = ({
...
-                                    home
-                                />
-                            </div>
-                        </div>
-                        <div className={css.blogPost_title_right}>
-                            <h1 className={`font_style__H1 ${css.blogPost_section_headline}`}>{post.title}</h1>
-                            <div className={css.blogPost_rectangle} />
-                        </div>
-                    </section>
+                    <Intro pathname={location.pathname} home title={post.title} />
```
Comment (eugene-matvejev) [inline src/templates/blog-post-template.jsx]: 👍

File: src/templates/blog-post-template.data-provider.js [hunk 6]
```diff
@@ -44,7 +44,7 @@ export default
                 sizes: '1',
             },
         },
-        description: '{{data.contentfulBlogPost.description.description}}',
+        description: { description: '{{data.contentfulBlogPost.description.description}}' },
```
Comment (eugene-matvejev) [inline src/templates/blog-post-template.data-provider.js]: 👍

File: src/components/intro.module.css [hunk 7]
```diff
@@ -1,14 +1,72 @@
...
+
+  .intro__section__rectangle {
+    margin-top: 64px;
+    margin-bottom: 0px;
+    width: 206px;
+    height: 10px;
+  }
+  .intro__section__headline {
+    overflow: hidden;
+    -webkit-line-clamp: 2;
```
Comment (eugene-matvejev) [inline src/components/intro.module.css]: is there is any better way to use it like generic? or it is only for webkit? :)

Comment (eugene-matvejev) [reply src/components/intro.module.css]: https://css-tricks.com/almanac/properties/l/line-clamp/

**Pull request merged**